item-spec/item-spec.md

```diff
@@ -166,6 +166,12 @@ or streamed. It is allowed to add additional fields.
 | type        | string    | [Media type](#media-types) of the asset. |
 | roles       | \[string] | The [semantic roles](#asset-role-types) of the asset, similar to the use of `rel` in links. |
 
+In addition to the fields above, [additional fields](#additional-fields) may be added to the assets. (See [Additional Fields for Assets](#additional-fields-for-assets))
+
+* [STAC Common Metadata](common-metadata.md#stac-common-metadata) - A list of fields commonly used throughout all domains.
+* [Content Extensions](../extensions/README.md#list-of-content-extensions) - Domain-specific fields such as EO, SAR and point clouds.
+* [Custom Extensions](../extensions/README.md#extending-stac) - It is generally allowed to add custom fields.
```

Comment:
I'd move these lists of links (common, content extensions, custom extensions) down to the section on 'additional fields for assets', like only present these links *after* we emphasize you should be really selective and why it's not the best idea to use these. Right now it reads like 'use all of these', and only later says 'probably don't use them'.

Reply:
Well, maybe in this case just link to the "Addtional Fields" section anyway as it's already there for Item Properties.

Reply:
Ah yeah, I like that approach.